dev/Interop/CameraCaptureUI/CameraCaptureUI/CameraCaptureUI2.idl

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.Windows.Media.Capture
+{
+    [contractversion(1)]
+    apicontract CameraCaptureUIContract {};
```

Comment:
Better to add the contract "CameraCaptureUIContract" in specs/WinRT/WinRTAPIContracts.md as well.

Reply:
Created a task to track this internally.